dojo/tools/appcheck_web_application_scanner/parser.py

```diff
@@ -0,0 +1,87 @@
+import importlib.util
+import inspect
+import json
+import logging
+import os
+from typing import Any
+
+from dojo.tools.appcheck_web_application_scanner.engines.base import BaseEngineParser
+
+LOGGER = logging.getLogger(__name__)
+
+
+def load_scanning_engine_parsers() -> dict[str, BaseEngineParser]:
```

Comment:
Is there a particular reason to do this dynamically rather than just importing the known engines explicitly? Might encourage people to contribute additional engines (if any) rather than just keeping them to themselves 😉

Reply:
Forward-planning laziness. 😂 Excellent point about encouraging contributions, I've fixed it. Thanks!